app/views/users/_user.html.erb

```diff
@@ -3,5 +3,6 @@
   <span class="profile-link">
       <%= link_to 'See Profile |',  user_path(user), class: 'profile-link' %>
   </span>
-  <%= friend_request(user) %>
+  <p>You have already invited this person to be your friend.</p>
```

Comment:
- [ ] This text is still appearing even if I just created the accounts and I haven't yet sent a friend request, please double-check the logic here and display an 'Add friend" button (There should exist 2 add friend buttons, one directly in the user's profile of each person and one in your "All users" view, you are missing the second one):
![image](https://user-images.githubusercontent.com/38522578/133861075-3574e059-4ed1-4220-b90c-228ed53e2048.png)
The other issues mentioned by the previous CR seems to be fixed now and friendships can be accepted and declined accordingly, great job :)

[OPTIONAL]:

- [ ] I would highly recommend any kind of user feedback in your "friend requests" section like "No friend requests found/received yet" rather than having a blank page:
![image](https://user-images.githubusercontent.com/38522578/133861252-a338c2ba-a901-435f-9112-779b83cc797b.png)